electrochemistry.ttl

```diff
@@ -2947,34 +3153,48 @@ charge passed at infinite time, D is the diffusion coefficient of the electroact
+                                                       emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "chronopotentiometry where the applied current is changed in steps"@en ;
                                                        skos:prefLabel "StepChronopotentiometry"@en .
 
 [ rdf:type owl:Axiom ;
    owl:annotatedSource :electrochemistry_7416a831_ba72_447c_bf83_3989ba2dfe37 ;
    owl:annotatedProperty emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 ;
-   owl:annotatedTarget "Chronopotentiometry where the applied current is changed in steps."@en ;
+   owl:annotatedTarget "chronopotentiometry where the applied current is changed in steps"@en ;
    dcterms:source "J. M. Pingarrón et al., Terminology of electrochemical methods of analysis (IUPAC Recommendations 2019), Pure and Applied Chemistry, 4, 92, 2020, 641-694. https://doi.org/10.1515/pac-2018-0109."
  ] .
 
 
 ###  http://emmo.info/electrochemistry#electrochemistry_74459386_875c_4303_b774_60125b599d06
 :electrochemistry_74459386_875c_4303_b774_60125b599d06 rdf:type owl:Class ;
-                                                       rdfs:subClassOf :electrochemistry_1aec4cc0_82d5_4042_a657_ed7fe291c3d8 ;
+                                                       rdfs:subClassOf :electrochemistry_1aec4cc0_82d5_4042_a657_ed7fe291c3d8 ,
+                                                                       [ rdf:type owl:Restriction ;
+                                                                         owl:onProperty emmo:EMMO_0aa934ee_1ad4_4345_8a7f_bc73ec67c7e5 ;
+                                                                         owl:someValuesFrom emmo:EMMO_43003c86_9d15_433b_9789_ee2940920656
+                                                                       ] ,
+                                                                       [ rdf:type owl:Restriction ;
+                                                                         owl:onProperty emmo:EMMO_0aa934ee_1ad4_4345_8a7f_bc73ec67c7e5 ;
+                                                                         owl:someValuesFrom emmo:EMMO_cd2cd0de_e0cc_4ef1_b27e_2e88db027bac
+                                                                       ] ,
+                                                                       [ rdf:type owl:Restriction ;
+                                                                         owl:onProperty emmo:EMMO_0aa934ee_1ad4_4345_8a7f_bc73ec67c7e5 ;
+                                                                         owl:someValuesFrom emmo:EMMO_e4de48b1_dabb_4490_ac2b_040f926c64f0
+                                                                       ] ;
                                                        emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "a soft pouch case that is described by its length, width, and height"@en ;
                                                        skos:prefLabel "PouchCase"@en .
```

Comment:
We might consider moving this and related terms to domain-battery

Reply:
I left it in domain-electrochemistry because it might be useful also for fundamental electrochemical activities. And I didn't want to split up like cases for CoinCell and PouchCell...